ocaml/xapi/xapi_pool_update.ml

```diff
@@ -275,6 +283,7 @@ let parse_update_info xml =
       key = Filename.basename key;
       installation_size = installation_size;
       after_apply_guidance = guidance;
+      enforce_homogeneity = enforce_homogeneity;
```

Comment:
That's my favourite style too :smile: (learn it from elm-lang: http://elm-lang.org/docs/style-guide )